.gitignore

```diff
@@ -1,2 +1,3 @@
 *~
 *.bak
+civicrm_constants.local.php
```

Comment:
Okay, I probably should have been more selective as I created my PR. Obviously this should not be merged.

Reply:
@monishdeb Could you review in the next couple of days and merge as appropriate. Thanks, @mlutfy !!

Reply:
To clarify: `civicrm_constants.local.php` is a new file that makes it possible to override defaults in `civicrm_constants.php` (to change tax names / to translate them).